python/lang/correctness/return-in-init.py

```diff
@@ -1,43 +1,46 @@
 class A:
-
     def __init__(a, b, c):
         # ruleid:return-in-init
         return A(a, b, c)
 
-class A:
-    print ('hello')
+
+class B:
```

Comment:
This case wasn't caught previously because we were only detecting `return ...` not `return`. And another bug mentioned below.